### DISCUSSION TITLE
Home-Assistant Blog

### DISCUSSION THREAD

**Discussion by DurgNomis-drol** [leikoilja/ha-google-home#173] [Show and tell]
Hey guys!

Our integration made it to the frontpage of Home-Assistant Blog! 🎉 

https://www.home-assistant.io/blog/2021/04/09/community-highlights/

It may have been me who submitted it a while ago. 😉

**Comment by leikoilja** [comment]
Hhaha, those are some awesome news, thanks team for an amazing work 💯 🔥 and @DurgNomis-drol for submitting it to the HA blog 💥  
Now we can post that link as an achievement on our linkedin profiles hahhaa 🚀 😄

**Comment by KapJI** [comment]
Wow, that's awesome! 🎉
